pynest/examples/eprop_plasticity/eprop_supervised_regression_sine-waves_bsshslm_2020.py

```diff
@@ -196,6 +196,10 @@
     "V_th": 0.03,  # mV, spike threshold membrane voltage
 }
 
+# factors from the original pseudo-derivative definition are incorporated into the parameters
+params_nrn_rec["gamma"] /= params_nrn_rec["V_th"]
+params_nrn_rec["beta"] /= np.abs(params_nrn_rec["V_th"])
```

Comment:
@akorgor  Why `np.abs` in one case?

Reply:
Because in the original definition `gamma * std::max( 0.0, 1.0 - beta * std::abs( ( v_m - v_th_adapt ) / V_th ) ) / V_th`  the `V_th` that got incorporated into the `beta` was inside the `std::abs()` whereas the `V_th` that got incorporated into the `gamma` was not.

Reply:
Thanks for the clarification. I think this may still be confusing to the reader. Is the `beta > 0` check in C++ an alternative?

Reply:
I agree that this could lead to confusion. The condition `beta > 0` is not required in general. The use of `abs()` is only necessary when reproducing the original case, as they have a rather non-standard definition of the surrogate gradient. Since `V_th > 0` in the original parameters, we could also omit the `abs()`, but in that case, the original case will only work as long as `V_th` is not set to negative values. Perhaps we could add a comment to clarify the use of abs() instead?

Reply:
Yes, a little comment might be helpful here.

Reply:
Ok, I will upload a change.